zio-http/src/main/scala/zio/http/api/internal/RichTextCodec.scala

```diff
@@ -72,6 +73,17 @@ sealed trait RichTextCodec[A] { self =>
    */
   final def describe: Doc = Doc.p(Doc.Span.code(RichTextCodec.describe(self)))
 
+  /**
+   * Tags the codec with a label used in the documentation
+   */
+  final def @@(label: String): RichTextCodec.Tagged[A] = RichTextCodec.Tagged(label, self)
```

Comment:
Let's use `??` and `?!` as the symbolic operators. Also, we should have named versions: `tagged` and `taggedUnexplained` or something.